.github/workflows/js-pull-request.yml

```diff
@@ -0,0 +1,29 @@
+name: Pull Request (JS)
+
+on:
+  pull_request:
+    paths:
+      - js/*
+  push:
+    branches: [master]
+    paths:
```

Comment:
what is this

Reply:
this section is for running the tests in the `cli` package, but only when those files change (don't need to run these tests for rust-related changes, for example). this is not running on this PR because `pull_request` only works in the context of the source repository (`metaplex`) and until it reaches the default branch it won't run on forked PRs. if this lands, though, it'll make sure we're running tests whenever someone makes a change to the CLI. 

you can see this test running successfully [here](https://github.com/jnwng/metaplex/pull/1/checks?check_run_id=3708789793)